src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -1698,6 +1710,22 @@ public static function createNote(array $item): array
 		return $data;
 	}
 
+	/**
+	 * Replace the share block with a link
+	 *
+	 * @param string $body
+	 * @return string
+	 */
+	private static function ReplaceSharedData(string $body): string
```

Comment:
Please use `snakeCase` for method names.
```suggestion
	private static function replaceSharedData(string $body): string
```